sormas-ui/src/main/java/de/symeda/sormas/ui/utils/UserField.java

```diff
@@ -67,6 +71,7 @@ protected Component initContent() {
 
 		addValueChangeListener(c -> {
 			userCombo.setValue(c.getProperty().getValue());
+			parentPseudonymizedFlag = parentPseudonymizedSupplier.get();
```

Comment:
same as above

Reply:
removed and moved in the pop-up window code